ansible_collections/arista/avd/examples/dual-dc-l3ls/README.md

```diff
@@ -0,0 +1,609 @@
+
+The following section covers the L3 leaf switches. Significantly more settings need to be set compared to the spine switches:
+
+```yaml title="DC2.yml"
+l3leaf:
+  defaults:
+    platform: vEOS-lab # (1)!
+    loopback_ipv4_pool: 10.255.128.0/27 # (2)!
+    loopback_ipv4_offset: 2 # (3)!
+    vtep_loopback_ipv4_pool: 10.255.129.0/27 # (4)!
+    uplink_interfaces: ['Ethernet1', 'Ethernet2'] # (5)!
+    uplink_switches: ['dc2-spine1', 'dc2-spine2'] # (6)!
+    uplink_ipv4_pool: 10.255.255.64/26 # (7)!
+    mlag_interfaces: ['Ethernet3', 'Ethernet4'] # (8)!
+    mlag_peer_ipv4_pool: 10.255.129.64/27 # (9)!
+    mlag_peer_l3_ipv4_pool: 10.255.129.96/27 # (10)!
+    bgp_defaults:
+      - no bgp default ipv4-unicast
+      - distance bgp 20 200 200
+      - graceful-restart restart-time 300
+      - graceful-restart
+    virtual_router_mac_address: 00:1c:73:00:00:99 # (11)!
+    spanning_tree_priority: 4096 # (12)!
+    spanning_tree_mode: mstp # (13)!
+
+  node_groups: # (14)!
+    DC2_L3_LEAF1:
+      bgp_as: 65201 # (15)!
+      nodes:
+        dc2-leaf1a:
+          id: 11
+          mgmt_ip: 172.16.1.201/24
+          uplink_switch_interfaces: # (16)!
+            - Ethernet1
+            - Ethernet1
+        dc2-leaf1b:
+          id: 12
+          mgmt_ip: 172.16.1.202/24
+          uplink_switch_interfaces:
+            - Ethernet2
+            - Ethernet2
+
+    DC2_L3_LEAF2:
+      bgp_as: 65202
+      evpn_gateway: # (17)!
+        evpn_l2:
+          enabled: true
+        evpn_l3:
+          enabled: true
+          inter_domain: true
+      nodes:
+        dc2-leaf2a:
+          id: 13
+          mgmt_ip: 172.16.1.203/24
+          uplink_switch_interfaces:
+            - Ethernet3
+            - Ethernet3
+          evpn_gateway:
+            remote_peers: # (18)!
+              - hostname: dc1-leaf2a
+        dc2-leaf2b:
+          id: 14
+          mgmt_ip: 172.16.1.204/24
+          uplink_switch_interfaces:
+            - Ethernet4
+            - Ethernet4
+          evpn_gateway:
+            remote_peers:
+              - hostname: dc1-leaf2b
+```
+
+1. `platform` references default settings defined in AVD specific to certain switch platforms.
+2. `loopback_ipv4_pool` defines the IP scope from which AVD assigns IPv4 addresses for Loopback0. Please note that this IP pool is identical to the one used for the spine switches in this example. To avoid setting the same IP addresses for several devices, we define the option `loopback_ipv4_offset`.
+3. `loopback_ipv4_offset` offsets all assigned loopback IP addresses counting from the beginning of the IP scope. This is required when the same IP pool is used for two different node_types (like spine and l3leaf in this example) to avoid overlapping IPs. The offset is "2" in this case because each spine switch uses one loopback address.
```

Comment:
```suggestion
3. `loopback_ipv4_offset` offsets all assigned loopback IP addresses counting from the beginning of the IP scope. This is required when the same IP pool is used for two different node_types (like spine and l3leaf in this example) to avoid overlapping IPs. For example, the offset is "2" because each spine switch uses one loopback address.
```